users/templates/base.html

```diff
@@ -39,98 +40,64 @@
-                    <span>{% trans 'Home' %}</span>
-                </a>
-                {% endif %}
             </li>
             <li>
-                {% if request.path|startswith:account_url or request.path|startswith:password_change_url %}
-                <a class="active" href="{{ account_url }}">
-                    <img src="{% static 'users/images/white/iko-01.png' %}">
-                    <span>{% trans 'Account' %}</span>
-                </a>
-                {% else %}
-                <a href="{{ account_url }}">
-                    <img src="{% static 'users/images/grey/ico-01.png' %}">
-                    <span>{% trans 'Account' %}</span>
+                <a href="{{ account_url }}"
+                   {% if request.path|startswith:account_url or request.path|startswith:password_change_url %}
+                   class="active"{% endif %}>
+                    <span class="link-icon"><i class="fa fa-user"></i></span>
+                    <span class="link-text">{% trans 'Account' %}</span>
                 </a>
-                {% endif %}
             </li>
             {% if user.user_type == admin %}
             <li>
-                {% if request.path|startswith:custom_users_list_url or request.path|startswith:'/employees/reports/author/' %}
-                <a class="active" href="{{ custom_users_list_url }}">
-                    <img src="{% static 'users/images/white/iko-03.png' %}">
-                    <span>{% trans 'Employees' %}</span>
-                </a>
-                {% else %}
-                <a href="{{ custom_users_list_url }}">
-                    <img src="{% static 'users/images/grey/ico-03.png' %}">
-                    <span>{% trans 'Employees' %}</span>
+                <a href="{{ custom_users_list_url }}"
+                   {% if request.path|startswith:custom_users_list_url or request.path|startswith:'/employees/reports/author/' %}
+                    class="active" {% endif %}>
+                    <span class="link-icon"><i class="fa fa-users"></i></span>
+                    <span class="link-text">{% trans 'Employees' %}</span>
                 </a>
-                {% endif %}
             </li>
             {% endif %}
             {% if user.user_type == manager or user.user_type == admin %}
             <li>
-                {% if request.path|startswith:custom_users_notifications %}
-                <a class="active" href="{{ custom_users_notifications }}">
-                    <img src="{% static 'users/images/white/notifications.png' %}">
-                    <span>{% trans 'Notifications' %}</span>
+                <a href="{{ custom_users_notifications }}" {% if request.path|startswith:custom_users_notifications %}
+                   class="active"{% endif %}>
+                    <span class="link-icon"><i class="fa fa-bell"></i></span>
+                    <span class="link-text">{% trans 'Notifications' %}</span>
```

Comment:
Maybe we could make all this code in `li` tags in partial? We would be able to pass three variables like `url`` string with icon name` and `string to trans`. I am not sure about our `if syntax` because a few of them have `or condition` but maybe we could create some template tag to check this in one condition.

It is just my small suggestion ;).